.pre-commit-config.yaml

```
@@ -1,26 +1,16 @@
-exclude: _vendor|vendored
 repos:
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.9.9
+  rev: v0.14.7
```

```suggestion
  rev: v0.14.6
```

I want to keep ruff a little outdated to allow work on automatic bumping of pre-commit config in this repository.